h/views/admin/groups.py

```diff
@@ -71,10 +71,16 @@ def post(self):
         def on_success(appstruct):
             """Create a group on successful validation of POSTed form data."""
 
-            organization = self.organizations[appstruct["organization"]]
+            organization = self.organizations.get(appstruct["organization"])
```

Comment:
Fix the create/edit group admin page to not assume that `appstruct` (i.e. the submitted form fields) contains an organization whose value matches one of the keys in `self.organizations`. If the group has no organization then `appstruct["organization"]` will be `None` and the local variable `organization` will end up being `None`